mne/beamformer/tests/test_lcmv.py

```diff
@@ -328,8 +294,8 @@ def test_make_lcmv(tmpdir, reg, proj):
     rank = 17 if proj else 20
     assert 'LCMV' in repr(filters)
     assert 'unknown subject' not in repr(filters)
-    assert '484' in repr(filters)
-    assert '20' in repr(filters)
+    assert '4157 vert' in repr(filters)
```

Comment:
why did you have to change this?

Reply:
I pulled the sphere code to a separate test below, so the repr is for a different set of `filters` now (computed earlier on a different sec). I could have moved these lines to the sphere test instead unchanged but it made more sense to keep them with the primary test